contrib/completion/bash/docker

```diff
@@ -1441,6 +1445,99 @@ _docker_version() {
 	esac
 }
 
+_docker_volume_create() {
+	case "$prev" in
+		--driver|-d)
+			COMPREPLY=( $( compgen -W "local" -- "$cur" ) )
```

Comment:
I don't think that'd be nescessary; chances are big that they're not available on most installs (and there are far more plugins than those)

If we want completion for plugins, I think docker should expose an API to get a list of available/installed plugins